integration/test-integration/src/test/resources/apiYaml/ratelimit_api.yaml

```diff
@@ -0,0 +1,95 @@
+type: api
+version: v4.1.0
+data:
+  id: 2b547d7c-4f7b-4f31-ad6b-05b4960dcbf1
+  name: ratelimit
+  context: /ratelimit
+  version: 1.0.0
+  rateLimitLevel: API
+  rateLimitPolicy: $_5PerMin
```

Comment:
Shall we add an operation-level test as well?

Reply:
added